bin/bundler-audit

```diff
@@ -4,10 +4,12 @@ $LOAD_PATH.unshift(File.expand_path(File.join(File.dirname(__FILE__), "../lib"))
 
 require 'cc/engine/bundler_audit'
 
-def parse_engine_config(raw)
-  raw ? JSON.parse(raw) : {}
+engine_config = {}
+
+if File.exists?("/config.json")
+  engine_config = JSON.parse(File.read("/config.json"))
 end
```

Comment:
I like the second option - will implement that. Ruby doesn't make a new scope for if/else, so the third is a weird pattern to me.